tests/cpp/leds.cpp

```diff
@@ -1,4 +1,4 @@
-#include "mtrain.hpp"
```

Comment:
the header file is still called mtrain.hpp so it doesn't compile on my computer cause of this.